DependencyInjectionContainer/Enums/ResolveSource.cs

```diff
@@ -4,10 +4,12 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace DIFixture.TestClasses
+namespace DependencyInjectionContainer.Enums
 {
-    public interface IProgrammer
+    public enum ResolveSource
```

Comment:
Name looks like a verb (it does something) but it is more about traits. What about changing the name to some trait-like, for example, ResolveResourceKind ResolveStrategy or so